homeassistant/components/fitbit/api.py

```diff
@@ -120,12 +122,23 @@ async def async_get_latest_time_series(self, resource_type: str) -> dict[str, An
         def _time_series() -> dict[str, Any]:
             return cast(dict[str, Any], client.time_series(resource_type, period="7d"))
 
-        response: dict[str, Any] = await self._hass.async_add_executor_job(_time_series)
+        response: dict[str, Any] = await self._run(_time_series)
         _LOGGER.debug("time_series(%s)=%s", resource_type, response)
         key = resource_type.replace("/", "-")
         dated_results: list[dict[str, Any]] = response[key]
         return dated_results[-1]
 
+    async def _run(self, func: Callable[[], _T]) -> _T:
+        """Run client command."""
+        try:
+            return await self._hass.async_add_executor_job(func)
+        except HTTPUnauthorized as err:
+            _LOGGER.debug("Unauthorized error from fitbit API: %s", err)
```

Comment:
Is this just logged at debug level now to not spam the log? It will change with the update coordinator?

Reply:
Yeah, that was my current thinking. There is some logging happening in the config flow which I thought was useful to put it in context of what it was doing, then I thought the data update coordinator would handle this and could be smarter about how often it logs.